python/mlx/optimizers.py

```diff
@@ -140,6 +140,144 @@ def apply_single(
         return parameter - self.learning_rate * update
 
 
+class RMSprop(Optimizer):
+    r"""Implementation of the RMSprop optimizer [1].
+
+    .. math::
+
+        v_{t+1} &= \beta v_t + (1 - \beta) g_t^2 \\
+        w_{t+1} &= w_t - \lambda \frac{g_t}{\sqrt{v_{t+1} + \epsilon}}
+
+    [1]: Tieleman, T. and Hinton, G. 2012. Lecture 6.5-rmsprop, coursera: Neural networks for machine learning
+    """
+
+    def __init__(self, learning_rate: float, beta: float = 0.99, eps: float = 1e-8):
```

Comment:
Any reason you went with `beta` instead of `alphas` as in PyTorch? If not, let's just use `alpha` for the sake of consistency?

Reply:
Agreed and changed.